cfgov/unprocessed/css/apps/owning-a-home/brand-footer.less

```diff
@@ -0,0 +1,62 @@
+.brand-footer {
+    background-color: @gray-5;
```

Comment:
Double check your [property order](https://github.com/cfpb/development/blob/master/standards/css.md#property-order) here and throughout.

Reply:
Fixed.